cirq-core/cirq/ops/pauli_gates_test.py

```diff
@@ -220,3 +220,24 @@ def test_powers():
     assert isinstance(cirq.X**1, cirq.Pauli)
     assert isinstance(cirq.Y**1, cirq.Pauli)
     assert isinstance(cirq.Z**1, cirq.Pauli)
+
+
+@pytest.mark.parametrize('gate_cls', (cirq.XXPowGate, cirq.YYPowGate, cirq.ZZPowGate))
+@pytest.mark.parametrize('exponent', (0, 0.5, 0.75, 1, 1.5))
```

Comment:
should we test a few negative exponents here?

Reply:
done